src/banner/Banner.style.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components'
+
+import { color, componentSizes, space } from '../_utils/branding'
+import { TextDisplay1 } from '../typography/display1'
+import { TextTitle } from '../typography/title'
+
+export const StyledBanner = styled.section`
+  background-color: ${color.boostBlue};
+  padding: ${space.xxl} ${space.xl};
+  height: 19.625rem; //314px magic number
```

Comment:
Maybe this should be a `min-height` so that if the title is too long the banner can stretch. I'd favor pixels (height of the phone image + top margin)

Reply:
I have a tendency to use rem since when you update your font-size base in the browser the bloc keeps the proportion defined.

Reply:
As you want, it's only been used once in the app.

Also regarding the height, makes me think we don't know what it is supposed to look like on small devices. I imagine either a flex-wrap to flip the phone below the text, or we simply the phone (if someone is browsing the site on their phone, they'd not scan the code - or maybe to show someone else but that's a rare scenario)